Demo/matchedvalues.py

```diff
@@ -35,7 +35,7 @@
 def print_result(search_result):
     for n in range(len(search_result)):
         print("dn: %s" % search_result[n][0])
-        for attr in search_result[n][1].keys():
+        for attr in search_result[n][1]:
```

Comment:
This isn't covered in the test suite. I'd prefer not changing it until it has tests.

Reply:
No problem. I've reverted it.